Sources/VGSShowSDK/UIElements/VGSLabel/VGSLabel+Internal.swift

```diff
@@ -130,4 +181,10 @@ internal extension VGSLabel {
 	func resetToRawText() {
 		textFormattersContainer.resetAllFormatters()
 	}
+
+	/// Update placeholder.
+	func updatePlaceholder() {
```

Comment:
probably we should rename this func to smth more specific

Reply:
Resolved. Renamed to `resetAllMasks`